src/CachePlugin.php

```diff
@@ -357,6 +367,7 @@ private function configureOptions(OptionsResolver $resolver)
 
             return empty($matches);
         });
+        $resolver->setAllowedTypes('response_mutator', ['null', 'Http\Client\Common\Plugin\Cache\Mutator\ResponseMutator']);
```

Comment:
We could use the `::class` constant

Reply:
I agree, but I was following the style of the other lines in this method.

Reply:
lets use it here and we fix it elsewhere.

(but as saying below, i think this should be an array, while we are at it.